message.d.ts

```diff
@@ -671,11 +680,14 @@ export interface StickerSet {
   name: string;
   /** Sticker set title */
   title: string;
+  /** Type of stickers in the set */
+  sticker_type: "regular" | "mask" | "custom_emoji";
   /** True, if the sticker set contains animated stickers */
   is_animated: boolean;
   /** True, if the sticker set contains video stickers */
   is_video: boolean;
-  /** True, if the sticker set contains masks */
+  /** True, if the sticker set contains masks
+   * @deprecated Use `sticker_type` instead */
```

Comment:
Perhaps must be removed instead? I followed the pattern from https://github.com/grammyjs/types/blob/273bceb460c7b57799eb4050289c192e3e3a22b1/proxied.d.ts#L686 to deprecate.